README.md

```diff
@@ -2,73 +2,250 @@
 
 The best way to handle your search bar documentation with [MeiliSearch](https://github.com/meilisearch/MeiliSearch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+To integrate a search bar to your documentation and use this library, you need to scrap your website contents and to push it into a MeiliSearch instance.
+
+Hopefully, we provide all the tools needed and [guide you into these different steps](https://docs.meilisearch.com/resources/howtos/search_bar_for_docs.html) 🚀
+
+As a first introduction, you might only want to test this library without connecting it to your website.<br>
+You can do it by running the playground provided in this repository:
+
+```bash
+$ yarn install
+$ yarn serve
+```
+
+The data comes from MeiliSearch documentation.<br>
+Type `create an indxe` to live the MeiliSearch experience.
```

Comment:
```suggestion
Type `create an index` to live the MeiliSearch experience.
```

Reply:
I think this typo is on purpose

Reply:
if it is, it would be better to make it clear in some way, cause it can be easily mistaken for a real typo :)

Reply:
I want our users to test the typo tolerance, I'm going to make it clearer.